container/cli.py

```diff
@@ -77,6 +77,26 @@ def subcmd_common_parsers(self, parser, subparser, cmd):
                                    help=u'Specify a local path containing roles you want to '
                                         u'use in the Conductor.')
 
+        if cmd in ('deploy', 'push'):
```

Comment:
Is the reason to move these just to avoid duplication between deploy+push, or is there something I'm not seeing?